examples/advanced/quickxplain.py

```diff
@@ -0,0 +1,83 @@
+"""
+    CPMpy implementation of the QuickXplain algorithm by Junker.
+        Junker, Ulrich. "Preferred explanations and relaxations for over-constrained problems." AAAI-2004. 2004.
+"""
+
+import cpmpy as cp
+from cpmpy.transformations.get_variables import get_variables
+from cpmpy.transformations.normalize import toplevel_list
+
+
+def quickXplain(soft, hard=[], order=lambda x : 0, solver="ortools"):
+    """
+        Find a preferred minimal unsatisfiable subset of constraints
+        A partial order is imposed on the constraints using the `order` argument
+        Constraints with lower order are preferred to ones with higher order
+
+        :param: soft: soft constraints, list of expressions
+        :param: hard: hard constraints, optional, list of expressions
```

Comment:
Something about hard = background constraints that cannot be relaxed (B) and soft = conflict (C) (that can be relaxed)